packages/@aws-cdk/aws-lambda/package.json

```diff
@@ -91,6 +91,7 @@
     "@aws-cdk/aws-s3": "0.0.0",
     "@aws-cdk/aws-s3-assets": "0.0.0",
     "@aws-cdk/aws-sqs": "0.0.0",
+    "@aws-cdk/aws-sns": "0.0.0",
```

Comment:
This shouldn't be needed anymore, now that sns and sqs have moved to the other `aws-lambda-event-sources` package.